.github/workflows/tests_linux_x86_nvidia_gpu.yml

```diff
@@ -0,0 +1,107 @@
+name: Tests::Linux::x86_64::Nvidia_GPU
+on:
+  release:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  COVERAGE_FLAGS: "--cov=pennylane_lightning_kokkos --cov-report=term-missing --cov-report=xml:./coverage.xml --no-flaky-report -p no:warnings --tb=native" 
+  GCC_VERSION: 11
+  CI_CUDA_ARCH: 86
+
+concurrency:
+  group: gpu-test-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  cpptests:
+    runs-on:
+      - self-hosted
+      - ubuntu-22.04
+      - gpu
+
+    strategy:
+      max-parallel: 1
+
+    steps:
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Remove Ubuntu unattended upgrades
+        run: sudo apt-get remove -y -q unattended-upgrades
+
+      - name: Install required packages
+        run: |
+          sudo apt-get update && sudo apt-get -y -q install gcc-${{ env.GCC_VERSION }} g++-${{ env.GCC_VERSION }} gcovr lcov
+          python -m pip install ninja cmake~=3.24.3
+      - name: Checkout pennyLane-lightning-kokkos
+        uses: actions/checkout@v3
+
+      - name: Validate GPU version and installed compiler
+        run: |
+          nvidia-smi
+          /usr/local/cuda/bin/nvcc --version
+      - name: Build and run unit tests
+        run: |
+            cmake . -BBuild \
+              -DKokkos_ENABLE_CUDA=ON \
+              -DKokkos_ARCH_AMPERE86=ON \ 
+              -DPLKOKKOS_BUILD_TESTS=ON \
+              -DPLKOKKOS_ENABLE_SANITIZER=ON \
+              -DCMAKE_CXX_COMPILER="$(which g++-${{ env.GCC_VERSION }})" \
+              -G Ninja
+            cmake --build ./Build
+            ./Build/pennylane_lightning_kokkos/src/tests/runner_kokkos
+      - name: Cleanup
+        if: always()
+        run: |
+          rm -rf *
+          rm -rf .git
+          rm -rf .gitignore
+          rm -rf .github
+  pythontests:
+    runs-on:
+      - self-hosted
+      - ubuntu-22.04
+      - gpu
+
+    strategy:
+      max-parallel: 1
+
+    steps:
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Install required packages
+        run: |
+          python -m pip install ninja cmake~=3.24.3 pytest pytest-mock flaky pytest-cov
+          sudo apt-get update && sudo apt-get -y -q install gcc-${{ env.GCC_VERSION }} g++-${{ env.GCC_VERSION }} gcovr lcov
+          # Sync with latest master branches
+          python -m pip install git+https://github.com/PennyLaneAI/pennylane.git@master
+          python -m pip install --index-url https://test.pypi.org/simple/ pennylane-lightning --pre
+          
+      - name: Checkout pennyLane-lightning-kokkos
+        uses: actions/checkout@v3
+
+      - name: Build and install package
+        run: |
+          CXX=$(which g++-${{ env.GCC_VERSION }}) BACKEND="CUDA" ARCH="AMPERE86" python -m pip install -e . --verbose
```

Comment:
Like nvcc isn't on path here either. I'd say following what `lightning.gpu` does as with https://github.com/PennyLaneAI/pennylane-lightning-gpu/blob/2d44c8699cbaebb289466b1027e1f10b22dfe8e7/.github/workflows/tests_linux_x86.yml#L134 should work here.